testsuite/regressiontests/issue-211.sli

```diff
@@ -68,11 +68,6 @@ n2 d4 << /weight 1. >> Connect
 [g5 g5] [n1] Connect
 g5 n3 << /weight 1. >> Connect
 
-% device-device; but g5g5 should be ignored
```

Comment:
why did you remove this whole block? I think the connection g5 -> g4 is still valid, and actually created, so I would suggest just removing the first line (`[g5 g5] [d4 g5] /one_to_one Connect`) and keeping the others. This would imply that you need to remove one `[5 4 0]` from the `target_conn` list.